src/icalendar/caselessdict.py

```diff
@@ -63,9 +63,6 @@ def pop(self, key, default=None):
     def popitem(self):
         return super().popitem()
 
-    def has_key(self, key):
-        key = to_unicode(key)
-        return super().__contains__(key.upper())
```

Comment:
I would argue that before removing a method it should be deprecated...
Anyway, nice thing to have this :)
And yeah... a changelog is very much welcome especially when you remove stuff.